src/libraries/Native/Unix/System.Globalization.Native/pal_icushim.c

```diff
@@ -41,6 +106,42 @@ static int FindICULibs()
     return TRUE;
 }
 
+static int FindSymbolVersion(int majorVer, int minorVer, int subVer, char* symbolName, char* symbolVersion, char* suffix)
```

Comment:
nit: I believe ICU calls the 3rd part of the version number the "patch-level" version, which is sort of inline with sem ver which calls it just PATCH.
Though I think on OSX this is maybe called the compatibility version? Not sure how much it matters though..

Reply:
Thanks, @jefgen, I'll leave it as is just to keep to have less changes as part of this PR as it is just a simple nit.